.github/workflows/tests.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install $(grep black requirements-tests-py3.txt)
-      - run: black --check --diff scripts stdlib stubs tests
+      - run: black --target-version py310 --check --diff scripts stdlib stubs tests
```

Comment:
Python 3.10 for black is now specified in 5 different places. Do we need it anywhere else than in pyproject.toml?

Reply:
No idea! Let's find out!

Reply:
Looks like we don't!